DotNet-Platform.md

```diff
@@ -0,0 +1,40 @@
+![Logo][logo]
+
+# The .NET Platform 
+
+**.NET platform** is big and comprehensive collection of frameworks and libraries manager by [Microsoft](https://www.microsoft.com/net). In the beginning it could be quite daunting to get familiar with .Net ecosystem.
```

Comment:
> In the beginning it could be quite daunting to get familiar with .Net ecosystem.

Should be `.NET`, and not `.Net`. But more importantly, don't bring up negative connotation, at least without an addendum like you will try to make it easy for the reader to follow alone. This could just as easily ward off readers, especially if they are beginners.